File: server/Responses/Laboratorio.cs
namespace server.Responses
{
    public class LaboratorioRes : BaseRes
    {
        public required int Id { get; set; }
        public required string Nombre { get; set; }
        public required string Direccion { get; set; }
        public required int Telefono { get; set; }
        public required int IdProvincia { get; set; }
        public required string Provincia { get; set; }

    }
}

File: web/src/views/private/Laboratorio/validations/laboratorio.ts
import * as Yup from "yup";

export const laboratorioSchema = Yup.object({
  nombre: Yup.string().required("Nombre es requerido"),
  idProvincia: Yup.number().required("La provincia es requerida"),
  direccion: Yup.string().required("Direccion es requerido"),
  telefono: Yup.string().required("telefono es requerido"),
});

export interface LaboratorioForm
  extends Yup.InferType<typeof laboratorioSchema> {}
